### ISSUE TITLE
Plotting does not show the INN results

### ISSUE THREAD

**Issue by fa7sa** [heidelberg-hepml/CaloINN#2]
I tried running the command mentioned in the github page. It certainly runs the code and generates  some images. However, in those images I can see the plotting results for VAE_INN and Geant4 although the legend for the INN is shown. I am wondering to see the plotting for INN do I have to give any extra parameter? I could not find related things inside the code. I am adding an image for reference. 
[E_layer_3_dataset_1-photons.pdf](https://github.com/heidelberg-hepml/CaloINN/files/14654900/E_layer_3_dataset_1-photons.pdf)

Also, in the appendix section of the paper, it mentioned about the CALOGAN dataset. I could not find related codes for that. Would you mind giving some guidance in this regard?

**Comment by luigifvr** [comment]
Hi fa7sa,
The main branch will always train a full INN model. The evaluation script, called during training, is borrowed from the calochallenge and works as a standalone. In our implementation, we allow for two input hdf5 files to compare their histograms and classifiers.
If only one file were passed, as during training, it would have copied it, thus giving two overlapping lines. The last commit fixes this. If the second input is 'None', now only one line will appear.
Notice that the label are currently hardcoded in: ```evaluate_plotting_helper.py```

I will also include a param card to train a model for the caloGAN dataset. Then, you can start a run in the same way from the ```calogan_data``` branch.

Best regards,
Luigi

**Comment by fa7sa** [comment]
Thanks for adding that. Also, I want to know if I want to run VAE+INN for DS3 which branch should I follow? I was trying running CaloINN with DS3 which is not possible(mentioned in the paper too). Also would you mind sharing the number of parameters for VAE+INN (DS3)

**Comment by floern1** [comment]
Hello fa7sa,

I pushed the last version of the VAE+INN to the branch VAE+INN. Furthermore, I added parameter files for the four different datasets.

I will updated the readme in the next days to explain the hyperparameters in the params file.

However, the current setup should work, if you add the paths to the datasets in the params file.

About your question of the trainable parameters (in order photons, pions, set2, set3):

encoder:
7.426600000000000000e+06
8.261600000000000000e+06
4.796285000000000000e+07
9.973150000000000000e+07

decoder:
7.374868000000000000e+06
8.201033000000000000e+06
4.707829400000000000e+07
9.904370000000000000e+07

learnable_norm:
0.000000000000000000e+00
0.000000000000000000e+00
1.305000000000000000e+04
8.109000000000000000e+04
(Set one might be misleading since it contains a non-learnable norm)

INN:
9.464400000000000000e+05
9.642420000000000000e+05
1.302480000000000000e+06
5.752980000000000000e+06

Best regards
Florian
